app/templates/views/service-settings.html

```diff
@@ -108,9 +108,9 @@ <h2 class="heading-small">{{ _('Your service is in trial mode') }}</h2>
         
         {% if config["FF_SPIKE_SMS_DAILY_LIMIT"] %}
           {% call row() %}
-            {% set txt = _('Daily SMS message limit') %}
+            {% set txt = _('Daily text fragments limit') %}
             {{ text_field(txt)}}
-            {% set sms_daily_limit = _('{} notifications').format(current_service.sms_daily_limit | format_number) %}
+            {% set sms_daily_limit = _('{}').format(current_service.sms_daily_limit | format_number) %}
```

Comment:
This is a fun way to write `current_service.sms_daily_limit | format_number`!

What I had in mind was more along `_('{} fragments').format(current_service.sms_daily_limit | format_number)`

Translated as `"{} fragments"`

We should also edit the row about emails to read as `"{} emails", "{} courriels"` instead of `"{} notifications"`

Reply:
Whoops, was on auto-pilot there I think :)

So this: 

```suggestion
            {% set sms_daily_limit = _('{} fragments').format(current_service.sms_daily_limit | format_number) %}
```

For the part about emails, I think we should wait until we switch the meaning of that setting, since it currently is still an overall message limit, and not yet an email-only limit.